### PR TITLE
refactor: rename to Credo

### PR DESCRIPTION
**Pull request by karimStekelenburg**
## Todo
- [x] Freeze 0.4 versioned docs
- [x] Freeze 0.4 snippets
- [x] Rename all mentions of "Aries JavaScript" to "Credo"
- [x] Rename all mentions of "Aries Framework JavaScript" to "Credo"
- [x] Remove mentions of "AFJ"
- [x] Update repository links

## Follow-up PR
- [ ] Update logo
- [ ] Update social preview
- [ ] Edit home page and mention the project's history
- [ ] Add blog functionality
- [ ] Write and publish post about the rebranding

### CODE REVIEW

File: guides/ecosystem/index.md [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-2. [Aries Framework JavaScript Extensions](https://credo.js.org/guides/extensions)
+1. [Credo](https://credo.js.org/guides)
+2. [Credo Extensions](https://credo.js.org/guides/extensions)
 3. [Aries Mobile Agent React Native (Bifold)](https://github.com/hyperledger/aries-mobile-agent-react-native/blob/main/README.md)
 
-### Aries Framework JavaScript
+### Credo
 
-Aries Framework JavaScript (AFJ) is at the core of the Aries JavaScript ecosystem. AFJ provides all the functionality related to cryptography, storage, messaging and more that is required
+Aries Framework JavaScript (AFJ) is at the core of the Credo ecosystem. AFJ provides all the functionality related to cryptography, storage, messaging and more that is required
```
Comment (TimoGlastra) [inline guides/ecosystem/index.md:23]: I think the regex didn't work everywhere

```suggestion
Credo is at the core of the Credo ecosystem. Credo provides all the functionality related to cryptography, storage, messaging and more that is required
```

File: guides/ecosystem/index.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
 
 ## Contributing in the Aries JS Ecosystem
```
Comment (TimoGlastra) [inline guides/ecosystem/index.md:25]:
```suggestion
## Contributing in the Credo Ecosystem
```

File: guides/getting-started/set-up/index.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ This guide assumes you have followed the [Prerequisites](./prerequisites), and y
 
 :::caution
 
-Aries Framework JavaScript is still in **active development**, and as such some APIs are still experimental. **When using any experimental features, make sure to use an exact version of AFJ** (`0.4.0`) instead of a range (`^0.4.0`), to prevent accidental breaking changes. If you're not leveraging any experimental features, you can use a range (`^0.4.0`) to get the latest bugfixes and features.
+Credo is still in **active development**, and as such some APIs are still experimental. **When using any experimental features, make sure to use an exact version of AFJ** (`0.4.0`) instead of a range (`^0.4.0`), to prevent accidental breaking changes. If you're not leveraging any experimental features, you can use a range (`^0.4.0`) to get the latest bugfixes and features.
```
Comment (TimoGlastra) [inline guides/getting-started/set-up/index.md:14]:
```suggestion
Credo is still in **active development**, and as such some APIs are still experimental. **When using any experimental features, make sure to use an exact version of Credo** (`0.4.0`) instead of a range (`^0.4.0`), to prevent accidental breaking changes. If you're not leveraging any experimental features, you can use a range (`^0.4.0`) to get the latest bugfixes and features.
```

I think a few AFJ / Aries Framework JavaScript searches without Credo will find some additional mentions

**Review by TimoGlastra — COMMENTED** [review]
Just some small comments, see you weren't finished yet but these were all related to the items you marked as ✅

File: guides/ecosystem/index.md (hunk 1 above)
Comment (karimStekelenburg) [reply guides/ecosystem/index.md:23]: I have intentionally skipped it where the text needs rewriting. "Credo is at the core of Credo" doens't make a lot of sense. I'll rewrite those after everything is renamed.

File: docusaurus.config.js [hunk 4]
```diff
@@ -48,8 +48,11 @@ const config = {
           lastVersion: 'current',
           versions: {
             current: {
+              label: 'v0.5.x',
```
[the remark this answers is not in the document]
Comment (karimStekelenburg) [reply docusaurus.config.js:51]: I thought we would wait with publishing until 0.5.0 was released? Also, my idea was to only rename the current version's docs, and keep the previous versions as is. I think if we make it clear that there was a rename (e.g. by adding a banner to older docs), that may be the least confusing. Thoughts?

[the remark this answers is not in the document]
Comment (karimStekelenburg) [reply docusaurus.config.js:51]: @TimoGlastra I have marked 0.5 as unreleased and set the default version to 0.4. This means that 'current' is still 0.5, but users visiting for the first time (clean local storage) will be presented with 0.4.

In order to set 0.4 as the current version we would have to put the 0.5 in the `versioned_docs` directory (next to 0.3) and have 0.4 live in the 'main' directories, which would become a bit messy when 0.5 is released.

We may want to consider moving everything into `versioned_docs` at some point for clarity. Right now we have multiple directories in the root that belong to the current version. We could refactor this so we have a `versioned_docs` directory that looks like this:
- `current/`
- `0.4/`
- `0.3/`

[the remark this answers is not in the document]
Comment (karimStekelenburg) [reply docusaurus.config.js:51]: @TimoGlastra one more thing regarding the versions. I've only applied the renames to the `0.5` versions of the docs. Do you want me to apply those to `0.4` as well? Since we will release `0.5` pretty soon, I figured adding a banner informing visitors about the rename instead. I want to avoid confusing people with documentation texts that says Credo and install commands that say AFJ.

Resolving for now. Feel free to unresolve if you don't agree.

File: .github/settings.yml [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 repository:
   name: credo-ts-docs
-  description: Extension libraries for Aries Framework JavaScript
+  description: Extension libraries for Credo
```
Comment (TimoGlastra) [inline .github/settings.yml:7]:
```suggestion
  description: Documentation for Credo
```

File: README.md [hunk 6]
```diff
@@ -6,7 +6,7 @@
     height="250px"
   />
 </p>
-<h1 align="center"><b>Aries JavaScript Documentation</b></h1>
+<h1 align="center"><b>Credo Documentation</b></h1>
 <p align="center">
   <a
     href="https://raw.githubusercontent.com/hyperledger/aries-framework-javascript/main/LICENSE"
```
Comment (TimoGlastra) [inline README.md:12]: Outdated link (i think we should do a find al on aries-framework-javascript and hyperledger as well

File: README.md [hunk 7]
```diff
@@ -19,10 +19,10 @@
 <p align="center">
   <a href="#getting-started">Getting started</a> &nbsp;|&nbsp;
   <a href="#contributing">Contributing</a> &nbsp;|&nbsp;
-  <a href="#license">License</a> 
+  <a href="#license">License</a>
 </p>
 
-Aries JavaScipt is an ecosystem of self-sovereign identity development tools, with [Aries Framework JavaScipt](https://github.com/hyperledger/aries-framework-javascript) at the center. This documentation site serves to make it as easy as possible for developers of any level to get started with building self-sovereign identity solutions.
+Credo is an ecosystem of self-sovereign identity development tools, with [Credo](https://github.com/openwallet-foundation/credo-ts) at the center. This documentation site serves to make it as easy as possible for developers of any level to get started with building self-sovereign identity solutions.
```
Comment (TimoGlastra) [inline README.md:25]: Sentence doesn't make full sense anymore

File: docusaurus.config.js [hunk 8]
```diff
@@ -6,8 +6,8 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula')
 const remarkTabs = require('remark-docusaurus-tabs')
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'Aries JavaScript Documentation',
-  tagline: 'Everything you need to know about the Aries JavaScript ecosystem.',
+  title: 'Credo',
+  tagline: 'Everything you need to know about the Credo TS ecosystem.',
```
Comment (TimoGlastra) [inline docusaurus.config.js:10]:
```suggestion
  tagline: 'Everything you need to know about the Credo ecosystem.',
```

File: docusaurus.config.js [hunk 9]
```diff
@@ -108,11 +112,7 @@ const config = {
...
                 label: 'Discord',
-                href: 'https://discord.com/channels/905194001349627914/941708033434738768',
-              },
-              {
-                label: 'Hyperledger Aries Mailing List',
-                href: 'https://lists.hyperledger.org/g/aries',
+                href: 'https://discord.com/channels/1022962884864643214/1179453305856991263',
               },
               {
                 label: 'Working Group Call Notes',
```
Comment (TimoGlastra) [inline docusaurus.config.js:118]: Should point to wiki in github

Comment (TimoGlastra) [reply docusaurus.config.js:118]: Also working group calendar can be removed, we have the wiki that lists both (line below)

File: guides/concepts/agents.md [hunk 10]
```diff
@@ -1,41 +1,41 @@
 # Agents
 
-When working with any Aries implementation, you will interact with an Aries
-agent. This will be either directly or via a REST API, like [the Aries
+When working with any Credo implementation, you will interact with an Credo
+agent. This will be either directly or via a REST API, like [the Credo
 framework REST
```
Comment (TimoGlastra) [inline guides/concepts/agents.md:5]: I'd remove framework -- just Credo REST Api

File: guides/extensions/index.md [hunk 11]
```diff
@@ -2,15 +2,15 @@ import DocCardList from '@theme/DocCardList';
...
-| [`@aries-framework/push-notifications`](https://www.npmjs.com/package/@aries-framework/push-notifications) | ![@aries-framework/push-notifications version](https://img.shields.io/npm/v/@aries-framework/push-notifications) | Push notification plugin for Aries Framework JavaScript |
+| Package                                                                                                    | Version                                                                                                          | Description                                          |
+| ---------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| [`@aries-framework/redux-store`](https://www.npmjs.com/package/@aries-framework/redux-store)               | ![@aries-framework/redux-store version](https://img.shields.io/npm/v/@aries-framework/redux-store)               | Redux toolkit wrapper around Credo                   |
```
Comment (TimoGlastra) [inline guides/extensions/index.md:11]: I think we can also update the package names already . Or do you want to do it in a separate pr

File: guides/tutorials/agent-config/logging.md [hunk 12]
```diff
@@ -84,4 +84,4 @@ agentDependencies.indy.setLogger((level, target, message, modulePath, file, line
 > WARNING: You can only set the logger once. Call indy_set_default_logger, indy_set_logger, not both. Once it's been set, libindy won't let you change it.
 
 You can also set the environment variable `RUST_LOG` to log at specified log levels.
-See https://crates.io/crates/env_logger for more information.
+See <https://crates.io/crates/env_logger> for more information.
```
Comment (TimoGlastra) [inline guides/tutorials/agent-config/logging.md:87]: Intentional?

File: guides/tutorials/agent-config/index.md [hunk 13]
```diff
@@ -208,7 +208,7 @@ connectionImageUrl: 'https://picsum.photos/200'
 
 Whether the storage should automatically be updated when a newer version of
 [Aries Framework
```
Comment (TimoGlastra) [inline guides/tutorials/agent-config/index.md:210]: Still AFJ

**Review by TimoGlastra — APPROVED** [review]
Some small nits, but looks good! Going to merge already, can you do the other ones in a follow up?
